index.html

```diff
@@ -28,9 +28,10 @@
         <link href=https://web-experiments.lab.hum.uu.nl/jspsych/uil-utils/0.1/fonts.css rel="stylesheet" type="text/css"/>
 
         <!-- Uil OTS libraries -->
-        <script type="module" src="https://web-experiments.lab.hum.uu.nl/jspsych/uil-utils/1.0/jspsych-uil-utils-import.js"></script>
+        <script type="module" src="https://web-experiments.lab.hum.uu.nl/jspsych/uil-utils/1.1/jspsych-uil-utils-import.js"></script>
 
          <!-- Uil OTS organised  -->
+        <script src="https://web-experiments.lab.hum.uu.nl/jspsych/uil-utils/1.1/jspsych-plugins/plugin-survey.js"></script>
```

Comment:
When we use our own survey plugin we can probably remove the plugins from jspsych on lines 19 - 22 of this file

Reply:
jspsych-survey-text and jspsych-survey-multi-select are still used for the feedback and consent screens